File: MicroservizioMindTools/MindTools/src/main/java/it/contrader/dto/ExperienceDTO.java
package it.contrader.dto;

import java.util.List;


//import it.contrader.model.User;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class ExperienceDTO {
	private int idExperience;
	//private User user;
	private int idPrincipi;
	private String commento;
	private String positivo;
	private String negativo;
	private int score;
	private List<String> secon;
	private byte[] imagen;

}

File: Front-end/src/app/dto/ExperienceDTO.ts
import { User } from '../models/User';

export class ExperienceDTO{
    idPrincipi: number;
    commento: string; 
    positivo: string; 
    negativo: string;
    score: number;
    secon: Array<string>;
    


    constructor( idPrincipi: number, commento: string, positivo: string, negativo: string, score: number,
        secon: Array<string>){
            this.commento = commento;
            this.idPrincipi = idPrincipi;
            this.negativo = negativo;
            this.positivo = positivo;
            this.score = score;
            this.secon = secon;
           
        }
}